components/select/style/index.less

```diff
@@ -248,7 +248,7 @@
     }
 
     .@{select-prefix-cls}-search__field {
-      border: 0;
+      border: 0 @border-style-base @border-color-base;
```

Comment:
hmm actually it should be just `border-width: 0;` because `border:0` equals to `border: 0px initial initial` which render a black rectangle and overwrites the default gray color.

should I do another PR for this small change?